src/gapstate.h

```diff
@@ -117,9 +117,11 @@ typedef struct GAPState {
     Obj ExecState;
 
     /* From opers.c */
+#if defined(HPCGAP)
```

Comment:
I normally do it -- but the rest of `gapstate.h` has `#if defined(HPCGAP)` -- I guess this is what @markuspf prefers ? I really don't mind either way, I just tried to match the surrounding code. Happy to change, but I'd first like to hear @markuspf, so I don't have to change it back again later ;-).